guides/common/modules/proc_configuring-compliance-policy-deployment-methods.adoc

```diff
@@ -0,0 +1,32 @@
+[id="configuring-compliance-policy-deployment-methods_{context}"]
+= Configuring Compliance Policy Deployment Methods
+
+Use the following procedure to configure {Project} for the method that you have selected to deploy compliance policies.
+
+.Procedure
+* For the Ansible deployment method:
+** Ensure that you import the `theforeman.foreman_scap_client` Ansible role.
+
++
+For more information, see {ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].
+
++
+Then you must assign the created policy and the `theforeman.foreman_scap_client` Ansible role to a host or host group.
+To trigger the deployment, you must run the Ansible role on the host or host group either manually, or set up a recurring job using remote execution for regular policy updates.
+For more information, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
+* For the Puppet deployment method:
+. Ensure Puppet is enabled.
```

Comment:
You're changing the list type from Unordered for Ansible to ordered for Puppet. Is that intentional? Also, some lines begin with "Ensure that you ..." and some with "Ensure to ...". Mabye also drop the word "must" as it's a procedure and users have to follow it anyway?

Reply:
Yes, that's because for Ansible, there's only a single step.